mods/ra/maps/red-zone/weapons.yaml

```diff
@@ -0,0 +1,153 @@
+90mmAA:
+	Inherits: 90mm
+	InvalidTargets: Ground, Water
```

Comment:
`Ground` and `Water` no longer stand for both terrain and actors, take a look at bleed's weapon definitions to update this accordingly (same for warheads).

Reply:
Changed it and removed some duplication from the inherited definitions as well.